### DISCUSSION TITLE
Querydsl and Jakarta Persistence (JPA 2.3)

### DISCUSSION THREAD

**Discussion by vargadan** [querydsl/querydsl#3456] [Q&A]
Hi There,

We have implemented a DTO-less mapping and querying api heavily leveraging QueryDSL. 
Yet, we would like to  use it with spring 6 and hibernate 6 which would require jakara persistence / JPA 2.3 support from qdsl. 
I wonder if there are any plans to upgrate the querydsl-jpa module dependencies from javax.persistence to jakarta.persistence?
Thank you and kind regards,
Daniel

**Comment by nyg** [comment]
From what I understand, it is already the case, you need to use the jakarta classifier for the maven dependency:

```xml
<dependency>
    <groupId>com.querydsl</groupId>
    <artifactId>querydsl-jpa</artifactId>
    <version>${querydsl.version}</version>
    <classifier>jakarta</classifier>
</dependency>
```

**Reply by rishq** [reply]
unfortenately isn't  work
